run-media-eoslive.mount

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Endless Live USB Storage
+ConditionKernelCommandLine=endless.image.device
+ConditionKernelCommandLine=endless.live_boot
+
+[Mount]
+What=/dev/mapper/endless-image-device
+Where=/run/media/eoslive
```

Comment:
As I lay awake in the middle of the night, I had a thought: what will the permissions on this directory be? We need it to be writable by all users on the system:

- `eos-installer` saves eos-diagnostics to it on error, and may run either as `gnome-initial-setup` or as a "proper" user
- We are using (a subdirectory of) it as `$KOLIBRI_HOME`, and we don't (in general) know the UID of the user(s) who will run Kolibri

Reply:
I don't recall the mount options from the top of my head, but in my tests it was world-writable (and the fs being exFAT, we only need to worry about the mountpoint permissions, which are defined at mount time).